### ISSUE TITLE
Review the language used in UI (forms, templates etc)

### ISSUE THREAD

**Issue by Greenheart** [teknik-eksjo/chronos#72]
I've seen some mixed English and Swedish and also some phrasings that could be improved.

Might not be the highest priority right now but definitely something we want to fix before our production-deployment

**Comment by hugolundin** [comment]
Yep, this really needs to be fixed. The Swedish language overall in templates and error messages needs to refined.

**Comment by Greenheart** [comment]
@hugolundin Did you start working on this earlier today? Or is my memory failing me? :tongue:

**Comment by hugolundin** [comment]
Not really. I just changed Epost to E-post in some places.

**Comment by Greenheart** [comment]
Haha okay :+1:  But this is not our current priority when I think about it

**Comment by hugolundin** [comment]
+1
